### PR TITLE
[Feature](profile)Support active_queries TVF

### PR DESCRIPTION
**Pull request by wangbo**
## Proposed changes
Refactor tvf ```queries``` to ```active_queries```, and support show query runtime audit(scanrows/scanbytes/cputime/peakmemory)

```
mysql [(none)]>select BeHost,BePort, QueryId,StartTime,QueryTimeMs,WorkloadGroupId, QueryCpuTimeMs,ScanRows,ScanBytes,BePeakMemoryBytes,CurrentUsedMemoryBytes,`Database`,FrontendInstance from active_queries();
+------------+--------+-----------------------------------+---------------------+-------------+-----------------+----------------+----------+-------------+-------------------+------------------------+----------+------------------+
| BeHost     | BePort | QueryId                           | StartTime           | QueryTimeMs | WorkloadGroupId | QueryCpuTimeMs | ScanRows | ScanBytes   | BePeakMemoryBytes | CurrentUsedMemoryBytes | Database | FrontendInstance |
+------------+--------+-----------------------------------+---------------------+-------------+-----------------+----------------+----------+-------------+-------------------+------------------------+----------+------------------+
| 10.11.12.13 |   6090 | 4140c46f114b40f5-8134022811432205 | 2024-01-16 14:45:36 |       64505 |              -1 |          42245 | 24675172 | 37036601856 |        2464667460 |             2436927577 |          | vm-123   |
+------------+--------+-----------------------------------+---------------------+-------------+-----------------+----------------+----------+-------------+-------------------+------------------------+----------+------------------+
1 row in set (0.00 sec)

mysql [(none)]>select BeHost,BePort, QueryId,StartTime,QueryTimeMs,WorkloadGroupId, QueryCpuTimeMs,ScanRows,ScanBytes,BePeakMemoryBytes,CurrentUsedMemoryBytes,`Database`,FrontendInstance from active_queries();
+------------+--------+-----------------------------------+---------------------+-------------+-----------------+----------------+----------+-------------+-------------------+------------------------+----------+------------------+
| BeHost     | BePort | QueryId                           | StartTime           | QueryTimeMs | WorkloadGroupId | QueryCpuTimeMs | ScanRows | ScanBytes   | BePeakMemoryBytes | CurrentUsedMemoryBytes | Database | FrontendInstance |
+------------+--------+-----------------------------------+---------------------+-------------+-----------------+----------------+----------+-------------+-------------------+------------------------+----------+------------------+
| 10.11.12.13 |   6090 | 4140c46f114b40f5-8134022811432205 | 2024-01-16 14:45:36 |       65313 |              -1 |          42245 | 24675172 | 37036601856 |        2464667460 |             2436927577 |          | vm-123   |
+------------+--------+-----------------------------------+---------------------+-------------+-----------------+----------------+----------+-------------+-------------------+------------------------+----------+------------------+
1 row in set (0.01 sec)
```

schema
```
mysql [(none)]>desc function active_queries();
+------------------------+--------+------+-------+---------+-------+
| Field                  | Type   | Null | Key   | Default | Extra |
+------------------------+--------+------+-------+---------+-------+
| BeHost                 | TEXT   | No   | false | NULL    | NONE  |
| BePort                 | BIGINT | No   | false | NULL    | NONE  |
| QueryId                | TEXT   | No   | false | NULL    | NONE  |
| StartTime              | TEXT   | No   | false | NULL    | NONE  |
| QueryTimeMs            | BIGINT | No   | false | NULL    | NONE  |
| WorkloadGroupId        | BIGINT | No   | false | NULL    | NONE  |
| QueryCpuTimeMs         | BIGINT | No   | false | NULL    | NONE  |
| ScanRows               | BIGINT | No   | false | NULL    | NONE  |
| ScanBytes              | BIGINT | No   | false | NULL    | NONE  |
| BePeakMemoryBytes      | BIGINT | No   | false | NULL    | NONE  |
| CurrentUsedMemoryBytes | BIGINT | No   | false | NULL    | NONE  |
| Database               | TEXT   | No   | false | NULL    | NONE  |
| FrontendInstance       | TEXT   | No   | false | NULL    | NONE  |
| Sql                    | TEXT   | No   | false | NULL    | NONE  |
+------------------------+--------+------+-------+---------+-------+
14 rows in set (0.00 sec)
```

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/table/Queries.java [hunk 1]
```diff
@@ -33,7 +33,7 @@
 public class Queries extends TableValuedFunction {
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/table/Queries.java:33]: also rename this class name, it should be same as the tvf function name.

**Comment by wangbo** [comment]
run buildall

**Comment by wangbo** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**